capture/configs/aes_sca.yaml

```diff
@@ -0,0 +1,45 @@
+target:
+  target_type: cw310
+  fpga_bitstream: "../objs/lowrisc_systems_chip_earlgrey_cw310_0.1.bit"
+  force_program_bitstream: False
+  fw_bin: "../objs/aes_serial_fpga_cw310.bin"
+  pll_frequency: 100000000
+  baudrate: 115200
+  output_len_bytes: 16
+husky:
+  pll_frequency: 100000000
+  target_clk_mult: 0.1
```

Comment:
re `target_clk_mult`: here it is a param for the scope. we dont use it to adjust the target. arguably it could make sense to have a param for the CW target FPGA to adjust clock. but we need to know the target clk in scope too to compute samples from cycles.